test/utils/testconfig/testconfig.go

```diff
@@ -0,0 +1,295 @@
+package testconfig
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"context"
+
+	ptpv1 "github.com/openshift/ptp-operator/api/v1"
+	testclient "github.com/openshift/ptp-operator/test/utils/client"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ConfigCheck                  = "CONFIGCHECK"
```

Comment:
Do you mind adding comments to exported constants?

Reply:
fixed